example/data/rules/L2_HR_PIXC_kinesisRule.json

```diff
@@ -0,0 +1,15 @@
+{
+  "collection": {
+    "name": "L2_HR_PIXC",
+    "version": "000"
+  },
+  "name": "L2_HR_PIXC_kinesisRule",
+  "provider": "PODAAC_SWOT",
+  "rule": {
+    "type": "kinesis",
+    "value": "arn:aws:kinesis:{{awsRegion}}:{{awsAccountId}}:stream/{{streamName}}",
+    "arn": "arn:aws:kinesis:{{awsRegion}}:{{awsAccountId}}:stream/{{streamName}}"
```

Comment:
rule.value is the kinesis stream that kinesis-consumer lambda listens to.
rule.arn is the EventSourceMappings id which is automatically added when the mapping is created.
The rule needs to be created after the kinesis stream is in place.  I don't see this in the PR.

Reply:
It's in a new file. The stream is created at the beginning of integration tests, and destroyed at the end.  
https://github.com/nasa/cumulus/pull/455/files#diff-f4a144d837fc6c0abc86b61b63973879R10